src/main/java/com/cascadebot/cascadebot/commands/core/HelpCommand.java

```diff
@@ -5,16 +5,29 @@
 
 package com.cascadebot.cascadebot.commands.core;
 
+import com.cascadebot.cascadebot.CascadeBot;
 import com.cascadebot.cascadebot.commandmeta.CommandContext;
 import com.cascadebot.cascadebot.commandmeta.CommandType;
 import com.cascadebot.cascadebot.commandmeta.ICommand;
+import com.cascadebot.cascadebot.messaging.MessagingObjects;
+import com.cascadebot.cascadebot.utils.StringsUtil;
+import com.cascadebot.cascadebot.utils.pagination.Page;
+import net.dv8tion.jda.core.EmbedBuilder;
 import net.dv8tion.jda.core.entities.Member;
 
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
 public class HelpCommand implements ICommand {
```

Comment:
Don't add this command at the moment, we need a lot more commands/structure before this

Reply:
oH oopsie I didn't mean to push that sorry ooOPPSIE

Reply:
how do i remove it from the pr?

Reply:
Delete the command :P We won't need it for a while so no point working on it